### PR TITLE
Remove New/Update/Remove MgUserAuthenticationMethod

### PR DESCRIPTION
**Pull request by andres-canello**
﻿

Fixes #

### Changes proposed in this pull request

-
-
-

### Other links

-
-
-

### CODE REVIEW

**Comment by andres-canello** [comment]
Hi team, this PR is to remove the following commands that point to inexistent endpoints. The operations can be performed using the authentication-method-specific commands (i.e.: Remove-MgUserAuthenticationFido2Method):
New-MgUserAuthenticationMethod
Update-MgUserAuthenticationMethod
Remove-MgUserAuthenticationMethod

Please kindly verify that the directive added does what I intended :).

File: src/Identity.SignIns/Identity.SignIns/readme.md [hunk 1]
```diff
@@ -59,6 +59,10 @@ directive:
       verb: New|Update
       subject: ^UserAuthenticationPasswordMethod$
     remove: true
+  - where:
+      verb: New|Update|Remove
+      subject: ^MgUserAuthenticationMethod$
```
Comment (peombwa) [inline src/Identity.SignIns/Identity.SignIns/readme.md:64]: Let's remove the prefix from the subject.
```suggestion
      subject: ^UserAuthenticationMethod$
```

**Comment by andres-canello** [comment]
Thanks @peombwa !

**Review by peombwa — APPROVED** [review]

**Comment by FaithOmbongi** [comment]
FYA @msewaweru - Please see @peombwa's response on how CSDL annotations help prevent the overgeneration of cmdlets and let's investigate how the Graph onboarding team can help streamline the downstream PS cmdlet generation process. Thanks.

> Thank you for the contribution!
> 
> The PR looks good! The workload should ideally apply the following OData capability annotations in the [metadata](https://graph.microsoft.com/v1.0/$metadata) for us not to generate the commands:
> 
> ```
> <Annotations Target="microsoft.graph.authentication/methods">
>     <Annotation Term="Org.OData.Capabilities.V1.InsertRestrictions">
>         <Record>
>             <PropertyValue Property="Insertable" Bool="false" />
>         </Record>
>     </Annotation>
>     <Annotation Term="Org.OData.Capabilities.V1.UpdateRestrictions">
>         <Record>
>             <PropertyValue Property="Updatable" Bool="false" />
>         </Record>
>     </Annotation>
>     <Annotation Term="Org.OData.Capabilities.V1.DeleteRestrictions">
>         <Record>
>             <PropertyValue Property="Deletable" Bool="false" />
>         </Record>
>     </Annotation>
> </Annotations>
> ```
> 
> We can merge this PR to remove the commands in PowerShell, but the long-term fix should happen in the CSDL metadata.

**Comment by FaithOmbongi** [comment]
Tagging @marabooy. Please see the response from Peter - hopefully, this is also part of the Capability Scanner work.

**Pull request merged**